site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -200,66 +205,78 @@ the page it's trying to inject scripts into. Host permissions can either be gran
 requesting them as part of your extension's manifest (see [`host_permissions`][33]) or temporarily
 via [activeTab][15].
 
-Below we'll look at an example that uses activeTab.
+Below we'll look at a couple different versions of an activeTab-based extension.
 
-```json/3-5
+```json/4-6
+//// manifest.json ////
 {
   "name": "My extension",
   ...
   "permissions": [
     "activeTab"
   ],
-  ...
+  "background": {
+    "service_worker": "background.js"
+  }
 }
 ```
 
-Content scripts can be injected as files.
+Content scripts can be injected as files…
 
 ```js
-chrome.runtime.onMessage.addListener((message, callback) => {
-  if (message == "runContentScript"){
-    chrome.scripting.executeScript({
-      file: 'contentScript.js'
-    });
-  }
+//// content-script.js ////
+document.body.style.backgroundColor = 'orange';
+```
+
+```js
+//// background.js ////
+chrome.action.onClicked.addListener((tab) => {
+  chrome.scripting.executeScript({
+    target: { tabId: tabId },
```

Comment:
super nit: you can write this as `{ tabId }`

Reply:
Wrong, `tabId` is undefined. Should be `{ tabId: tab.id }`.

Same mistake in the other added examples.

Reply:
Ah shucks, thanks for the catch.